src/test-run/commands/options.js

```diff
@@ -6,35 +6,35 @@
 import Assignable from '../../utils/assignable';
 
 import {
-    ActionNumberOptionError,
-    ActionPositiveNumberOptionError,
+    ActionIntegerNumberOptionError,
+    ActionPositiveIntegerNumberOptionError,
     ActionBooleanOptionError
 } from '../../errors/test-run';
 
 // Validators
-function number (option, val, positive) {
+function integerNumber (option, val, positive) {
     var valType   = typeof val;
-    var ErrorCtor = positive ? ActionPositiveNumberOptionError : ActionNumberOptionError;
+    var ErrorCtor = positive ? ActionPositiveIntegerNumberOptionError : ActionIntegerNumberOptionError;
 
     if (valType !== 'number')
         throw new ErrorCtor(option, valType);
 
-    if (isNaN(val))
-        throw new ErrorCtor(option, valType);
+    if (isNaN(val) || !Number.isInteger(val))
```

Comment:
OK, didn't noticed that. So it means also no generators/`awaits` on a client, very sad. 
I'll use that check, it follows ECMA's [isInteger](https://tc39.github.io/ecma262/#sec-number.isinteger) definition.